generators/entity/files.js

```diff
@@ -16,9 +17,142 @@ module.exports = {
     writeFiles
 };
 
-function writeFiles() {
+function writeFiles(){
     return {
+        deleteRemoteEntityPath: function () {
+            if(!this.remove) return;
+
+            if (_.isUndefined(this.microservicePath)) {
+                return;
+            }
+
+            this.fs.delete(this.destinationPath(this.jhipsterConfigDirectory + '/' + this.entityNameCapitalized + '.json'));
+        },
+
+        deleteDbFiles: function () {
+            if(!this.remove) return;
+
+            if (this.skipServer) return;
+
+            if (this.databaseType === 'sql') {
+                this.warning('Please update manually the liquibase to deal with data before drop the table');
```

Comment:
Why is this required. I think we can delete liquibase changelog files and remove the entry from master xml as well and then print a warning to ask user to update the DB manually or by running liquibase:diff goal after removing the entity